modules/python/init.zsh

```diff
@@ -46,6 +46,20 @@ if (( $? && $+commands[virtualenvwrapper.sh] )); then
   source "$commands[virtualenvwrapper.sh]"
 fi
 
+# Load PIP completion.
+if (( $+commands[pip] )); then
+  cache_file="${0:h}/cache.zsh"
```

Comment:
This should probably use $TMPDIR (though that might be set in a different module... I can't remember, so using mktemp might be best) and the name should probably be more specific.